src/config/config_def.c

```diff
@@ -547,6 +547,7 @@ const char __WT_CONFIG_CHOICE_chunkcache[] = "chunkcache";
 const char __WT_CONFIG_CHOICE_compact[] = "compact";
 const char __WT_CONFIG_CHOICE_compact_progress[] = "compact_progress";
 const char __WT_CONFIG_CHOICE_configuration[] = "configuration";
+const char __WT_CONFIG_CHOICE_control_point[] = "control_point";
```

Comment:
The changes to config_def.c are too big for me to confidently review.

Reply:
config_def.c is a generated file. It does not need review. The changes are due to the changes in dist/api_data.py.